README.md

```diff
@@ -173,14 +174,15 @@ wheels for `torch`, `torchvision`, and `torch_xla` at
 
 | Package | Cloud TPU VMs Wheel (XRT on Pod, Legacy Only) |
 | --- | ----------- |
-| torch_xla | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch_xla-2.0-cp38-cp38-linux_x86_64.whl` |
-| torch | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch-2.0-cp38-cp38-linux_x86_64.whl` |
-| torchvision | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torchvision-2.0-cp38-cp38-linux_x86_64.whl` |
+| torch_xla | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch_xla-2.1-cp38-cp38-linux_x86_64.whl` |
+| torch | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torch-2.1-cp38-cp38-linux_x86_64.whl` |
+| torchvision | `https://storage.googleapis.com/tpu-pytorch/wheels/xrt/torchvision-2.1-cp38-cp38-linux_x86_64.whl` |
 
 <br/>
 
 | Version | GPU Wheel + Python 3.8 |
 | --- | ----------- |
+| 2.1+ CUDA 11.8 | `https://storage.googleapis.com/tpu-pytorch/wheels/cuda/118/torch_xla-2.1-cp38-cp38-linux_x86_64.whl` |
```

Comment:
2.1 on cuda 11.8 wheel would be: `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/11.8/torch_xla-2.1.0-cp38-cp38-manylinux_2_28_x86_64.whl`

2.2 on cuda 12.1 wheel would be: `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.2.0-cp38-cp38-manylinux_2_28_x86_64.whl`

Reply:
Thanks!